KeepingYouAwake/Settings/KYABatterySettingsViewController/Base.lproj/KYABatterySettingsViewController.xib

```diff
@@ -17,7 +17,7 @@
             <rect key="frame" x="0.0" y="0.0" width="450" height="268"/>
             <autoresizingMask key="autoresizingMask"/>
             <subviews>
-                <textField horizontalHuggingPriority="251" verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="UO8-po-rbJ">
+                <textField focusRingType="none" horizontalHuggingPriority="251" verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="UO8-po-rbJ">
```

Comment:
Can you please also exclude any changes of this file from the pull request? It does not contain any necessary changes for localization.